packages/browser-integration-tests/suites/replay/compression/test.ts

```diff
@@ -27,10 +28,16 @@ sentryTest('replay recording should be compressed by default', async ({ getLocal
   const url = await getLocalTestPath({ testDir: __dirname });
 
   await page.goto(url);
-  const replayEvent0 = getReplayEvent(await reqPromise0);
+  await forceFlushReplay();
+
+  const req0 = await reqPromise0;
+
+  const replayEvent0 = getReplayEvent(req0);
   expect(replayEvent0).toEqual(getExpectedReplayEvent());
 
-  const snapshots = getFullRecordingSnapshots(await reqPromise0);
+  expect(replayEnvelopeIsCompressed(req0)).toEqual(true);
```

Comment:
I also added a check to actually _test_ that we are sending a compressed payload 😅